test/cli_run_memory_test.go

```diff
@@ -35,73 +35,88 @@ func (suite *PouchRunMemorySuite) TearDownTest(c *check.C) {
 
 // TestRunWithMemoryswap is to verify the valid running container
 // with --memory-swap
-func (suite *PouchRunMemorySuite) TestRunWithMemoryswap(c *check.C) {
+func (suite *PouchRunMemorySuite) TestRunWithMemoryswapAndKernelMemory(c *check.C) {
 	SkipIfFalse(c, environment.IsMemorySupport)
 	SkipIfFalse(c, environment.IsMemorySwapSupport)
 
-	cname := "TestRunWithMemoryswap"
-	m := 1024 * 1024
+	cname := "TestRunWithMemoryswapAndKernelMemory"
 	memory := "100m"
 	memSwap := "200m"
-	expected := 200 * m
-	sleep := "10000"
+	kernelMemory := "100m"
+	expectedMem := strconv.Itoa(100 * 1024 * 1024)       // 100 MB
+	expectedKernelMem := strconv.Itoa(1024 * 1024 * 100) // 100 MB
```

Comment:
I think kernel memory test should be single test

Reply:
Actually both are OK. My concern is that if we combine the test case, we can reduce the CI time. It is a tradeoff.